src/main/java/net/neoforged/neoforge/common/NeoForgeEventHandler.java

```diff
@@ -152,7 +153,7 @@ public void onCommandsRegister(RegisterCommandsEvent event) {
     public void onResourceReload(AddReloadListenerEvent event) {
         INSTANCE = new LootModifierManager();
         event.addListener(INSTANCE);
-        event.addListener(DATA_MAPS = new DataMapLoader(event.getConditionContext(), event.getRegistryAccess()));
+        event.addListenerBefore(VanillaReloadListenerTarget.RECIPES, DATA_MAPS = new DataMapLoader(event.getConditionContext(), event.getRegistryAccess()));
```

Comment:
A passing comment: this is the first in a long time that I've seen an assignment expression used in a method call.